setup.py

```diff
@@ -31,8 +32,8 @@
       version=versioneer.get_version(),
       cmdclass=versioneer.get_cmdclass(),
       description='Orbital parameters and astronomical computations in Python',
-      author='Martin Raspaud, Esben S. Nielsen',
-      author_email='martin.raspaud@smhi.se',
+      author='The Pytroll Team',
+      author_email='pytroll@googlegroups.com',
```

Comment:
Is this what we finally decided on? Or that slack channel thing @mraspaud talked about?

Reply:
No, it is just an incremental improvement

Reply:
We can wait till you/we have made your mind up in Satpy and then follow, ok?

Reply:
Yeah, leaving it as you have it is fine by me.